.github/workflows/test-action.yml

```diff
@@ -4,7 +4,10 @@ on: [pull_request]
 jobs:
   shallow:
     name: Spell Check with Typos
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
`macos-latest` is still x64, right?  To ensure we also cover arm, I think we'd need `macos-14`